web/Migrations/BricksAndHeartsDbContextModelSnapshot.cs

```diff
@@ -65,7 +65,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasKey("Id");
 
-                    b.ToTable("Landlord");
+                    b.ToTable("Landlord", (string)null);

```

Comment:
This shouldn't have changed without a migration, probably a merge issue

Reply:
I think I changed this back during the rebase, it seemed weird there had been a change here